client/src/components/InitialPromptCreationScreen/InitialPromptCreationScreen.jsx

```diff
@@ -40,12 +76,29 @@ class InitialPromptCreationScreen extends React.Component {
   }
 
   render() {
-    const { error } = this.state;
+    const {
+      noun, adjective1, adjective2, error,
+    } = this.state;
     return (
       <div className="InitialPromptCreationScreen">
-        Adjective, adjective, noun.
+        <h3>Enter the prompts for other players to draw</h3>
```

Comment:
Nit: `a prompt` singular?

Reply:
(samesies for other places with plural `prompts`)

Reply:
Haha I always think of Prompt as a trio of Prompts.  But yea I updated it to be consistent with our naming choice.